custom_functions.py

```diff
@@ -31,6 +31,13 @@ def generate_password():
     characters = string.ascii_letters + string.digits
     return ''.join(random.choice(characters) for _ in range(12))
 
+# Custom Action for Token: {{GENERATE_PASSWORD}}
+id = 0
+def generate_id():
+    global id
+    result = id + 1
+    id = result
+    return result
```

Comment:
Happy to see you making a custom function! But this one won't be necessary as the DB auto increments the primary key for us.